### ISSUE TITLE
Wrong case for clusterIP key in ServiceSpec object

### ISSUE THREAD

**Issue by antoine-gaillard** [cdk8s-team/cdk8s#620]
### Description of the bug:

Due to a regression probably in the jsii layer when installing the latest cdk8s-cli and generating a Service with python,  the `clusterIP` field of the service has an incorrect case.
We get `clusterIp` key instead of `clusterIP` (with a capital p)

### Reproduction Steps:

Here is the minimal code to reproduce

```
#!/usr/bin/env python3
from constructs import Construct
from cdk8s import App, Chart

from imports import k8s

class MyChart(Chart):
    def __init__(self, scope: Construct, ns: str):
        super().__init__(scope, ns)

        k8s.KubeService(self, 'service',
            spec=k8s.ServiceSpec(
                cluster_ip='None'))

app = App()
MyChart(app, "bug")
app.synth()
```

### Error Log:

The generated chart looks like this 

```
apiVersion: v1
kind: Service
metadata:
  name: bug-service-c81eac80
spec:
  clusterIp: None
```

and we should get this instead

```diff
apiVersion: v1
kind: Service
metadata:
  name: bug-service-c81eac80
spec:
---  clusterIp: None
+++  clusterIP: None
```

### Environment:

I used this Dockerfile to reproduce 

```
FROM node
RUN npm install -g cdk8s-cli@1.0.0-beta.26
RUN apt update && apt install -y --no-install-recommends python3-pip
RUN pip3 install cdk8s=="1.0.0b26"
COPY cdk8s.yaml /app/
WORKDIR /app
RUN cdk8s import --language python 
COPY main.py /app/
RUN python3 main.py
```

and `cdk8s.yaml` file

```
language: python
imports:
  - k8s
```

### Other:

--- 

This is :bug: Bug Report

**Comment by Chriscbr** [comment]
Good catch!

I think this should be fixed as soon as we upgrade our json2jsii dependency to v0.1.271 with the changes from https://github.com/cdklabs/json2jsii/pull/480.

**Comment by Chriscbr** [comment]
Fixed in cdk8s-cli v1.0.0-beta.30.
